.eslintrc.js

```diff
@@ -105,6 +105,8 @@ module.exports = {
         ],
       },
     ],
+    // fix for .tsx
+    "react/jsx-filename-extension": "off",
```

Comment:
What is the aim of turning this rule off? I think it should only apply for `jsx` files and we don't use them. For typescript files we want to force using `tsx` extension. Do we want the same for js. In any case can you extend the comment to explicate it more.

About the word explication I find the following article giving good insights https://mindlevelup.wordpress.com/2017/07/15/explication/. Sharing resources ;)